components/app/Calendar.vue

```diff
@@ -93,7 +90,85 @@ const onEventClick = (event: any, e: any) => {
           List
         </label>
       </div>
-    </div> -->
+    </div>
+    <vue-cal
+      v-if="selectedView === 'calendar'"
+      class="rounded-lg bg-white dark:bg-neutral-900 overflow-hidden shadow border border-gray-400/40"
+      today-button
+      small
+      :events-on-month-view="true"
+      :twelve-hour="true"
+      :events="group.items"
+      :start-week-on-sunday="true"
+      :disable-views="['years', 'year', 'day']"
+      :time-from="8 * 60"
+      :time-to="22 * 60"
+      :time-step="30"
+      :on-event-click="onEventClick"
+      :show-all-day-events="['short', true, false]"
+    >
+      <template #arrow-prev>
+        <Icon
+          class="w-8 h-8"
+          name="i-ph-arrow-left"
+        />
+      </template>
+      <template #arrow-next>
+        <Icon
+          class="w-8 h-8"
+          name="i-ph-arrow-right"
+        />
+      </template>
+    </vue-cal>
+
+    <!-- List View -->
+    <div
+      v-if="selectedView === 'list'"
+    >
+      <div
+        v-for="(events, month) in groupedEvents"
+        :key="month"
+        class="mb-8"
+      >
+        <h3 class="text-2xl text-center mb-8 font-bold tracking-tight text-gray-900 dark:text-white">
+          {{ month }}
+        </h3>
+        <div class="grid xl:grid-cols-2 gap-4">
+          <div
+            v-for="(event) in events"
+            :key="event.title"
+            class="border-2 border-gray-300 rounded mb-4 break-word"
```

Comment:
can we use the same border colour as other places? thanks

```suggestion
            class="border-2 border-gray-400/40 rounded mb-4 break-word"
```